src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -868,13 +872,13 @@ export default class ElementWrapper extends Wrapper {
 					? x`@set_dynamic_element_data(${this.node.tag_expr.manipulate(block)})`
 					: x`@set_attributes`;
 
-		block.chunks.hydrate.push(
-			b`${fn}(${this.var}, ${data});`
+		block.chunks.hydrate.unshift(
+			b`${fn}(${this.var}, ${this.element_data_name});`
 		);
 
 		if (this.has_dynamic_attribute) {
-			block.chunks.update.push(b`
-				${fn}(${this.var}, ${data} = @get_spread_update(${levels}, [
+			block.chunks.update.unshift(b`
```

Comment:
For example, test case `component-event-handler-contenteditable3`, `get_spread_update` is added at `ElementWrapper` class, and `set_data_maybe_contenteditable` is added at `MustacheTag` class.
And `MustacheTag` is executed earlier than `ElementWrapper`.

If `get_spread_update` runs after `set_data_maybe_contenteditable`, `set_data_maybe_contenteditable` runs using `div_data` that is before updating.

To avoid this situation, I use `unshift`.
I think there are other ways to solve this issue, but I think this is reasonable and anyway, we will dynamically change the compiler in Svelte4, so we don't need to be particular about this.

```js
set_attributes$(div$, div_data$ = get_spread_update$(div$_levels$, [dirty & /*spread*/ 2 && /*spread*/ ctx[1]]));
if (dirty & /*text*/ 1) set_data_maybe_contenteditable$(t$, /*text*/ ctx[0], div_data$['contenteditable']);
```